compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/text/ContextMenu.desktop.kt

```diff
@@ -40,7 +55,7 @@ internal actual fun ContextMenuArea(
     if (DesktopPlatform.Current == DesktopPlatform.MacOS) {
         OpenMenuAdjuster(state) { manager.contextMenuOpenAdjustment(it) }
     }
-    ContextMenuArea(manager.contextMenuItems(), state, content = content)
+    LocalTextContextMenu.current.Area(manager.actions, state, content)
```

Comment:
IDEA complains that "This foundation API is experimental and is likely to change or be removed in the future." and suggests to add an OptIn
The example runs fine, but maybe the build of a release lib might fail?

Reply:
Strange. Which version of Kotlin/IDEA do you use (mine are 2022.1.1, 1.7.10)? Code is valid, because we have `@file:OptIn(ExperimentalFoundationApi::class)` on the top of the file.

Reply:
I have the same versions. Perhaps it's some bug.